integration/Jenkinsfiles/docker-compose-kafka-2.0.0.yml

```diff
@@ -0,0 +1,33 @@
+version: '2'
+services:
+  zookeeper:
+    image: zookeeper:3.4.13
+
+  kafka:
+    image: wurstmeister/kafka:2.11-2.0.0
+    environment:
+      KAFKA_ADVERTISED_HOST_NAME: kafka
+      KAFKA_ADVERTISED_PORT: 9092
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 10
+      KAFKA_OFFSETS_RETENTION_MINUTES: 20160
+    links:
+      - zookeeper
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Do we need this reference to the docker daemon socket?  Is this image scheduling other images?

Reply:
This comes from the original docker-compose of https://github.com/wurstmeister/kafka-docker/blob/master/docker-compose-single-broker.yml I suppose this is needed to submit docker command within the image like getting logs or discovery no idea if this is mandatory.

Reply:
:+1: